### PR TITLE
remove scancode and group support

### PR DESCRIPTION
**Pull request by hesa**
The functionality removed was working but adding support for groups
requires more non-code work than expected and such support (groups)
need to be rewritten from scratch.

### CODE REVIEW

File: flict/flictlib/compatibility.py [hunk 1]
```diff
@@ -37,46 +35,21 @@ class CompatibilityStatus(Enum):
...
-        else:
-            self.scancode_licenses = None
-        self.license_groups = LicenseGroups(group_file)
 
     def supported_licenses(self):
         license_set = set(self.compat_matrix.supported_licenses())
-        if self.scancode_licenses is not None:
-            license_set = license_set.union(
-                self.scancode_licenses.supported_licenses())
         return list(license_set)
```
Comment (priv-kweihmann) [inline flict/flictlib/compatibility.py:44]: This can be merged with the remaining line above

File: flict/flictlib/compatibility.py [hunk 2]
```diff
@@ -263,22 +236,17 @@ def check_compatibilities(self, licenses, check_all=False):
             outer_licenses = list(licenses)
             #print("outer licenses: " + str(outer_licenses))
 
-        for license_a in outer_licenses:
-            #print("check for: " + str(license_a))
-            #lic_str = str(l_fmt) % license_a
+        for lic_a in outer_licenses:
+            #print("check for: " + str(lic_a))
```
Comment (priv-kweihmann) [inline flict/flictlib/compatibility.py]: Can we please get rid of this dead debug code

**Review by priv-kweihmann — CHANGES_REQUESTED** [review]
Minor things... but please fix the stuff mentioned in the comments.
BTW I think the docu needs a refresh too

File: flict/flictlib/format/text_format.py [hunk 3]
```diff
@@ -20,11 +20,7 @@ def format_support_licenses(self, compatibility):
 
         ret_str = ""
         for item in supported_licenses:
-            lic_group = compatibility.license_group(item)
-            if lic_group is not None:
-                ret_str += " " + str(item) + ": (" + lic_group + ")\n"
-            else:
-                ret_str += " " + str(item) + "\n"
+            ret_str += " " + str(item) + "\n"
```
Comment (jens-erdmann) [inline flict/flictlib/format/text_format.py:23]: This should at least be an f-string and might be even more simplified.

**Comment by priv-kweihmann** [comment]
Almost there @hesa - only Jens's f-string issue and we can merged that here

File: flict/flictlib/format/text_format.py (hunk 3 above)
Comment (hesa) [reply flict/flictlib/format/text_format.py:23]: I'll do a rewrite of the strings in entire file afterwards

Comment (priv-kweihmann) [reply flict/flictlib/format/text_format.py:23]: @jens-erdmann are you okay with that?

**Pull request merged**